src/tools/rustfmt/ci/integration.sh

```diff
@@ -42,8 +42,9 @@ function check_fmt_with_lib_tests {
 
 function check_fmt_base {
     local test_args="$1"
-    local build=$(cargo test $test_args 2>&1)
-    if [[ "$build" =~ "build failed" ]] || [[ "$build" =~ "test result: FAILED." ]]; then
+    local build
+    build=$(cargo test "$test_args" 2>&1)
+    if [[ "$build" =~ "build failed" ]] || [[ "$build" =~ test\ result\:\ FAILED\. ]]; then
```

Comment:
Two lints here: variables in double quote strings and the most important one is:

```
=~ test\ result\:\ FAILED\.
```

`=~` means it's a regex check and the lint prefers to not have strings to ensure it doesn't have variables which could contain characters that might be interpreted as regex special characters (like the `.` here). So instead, we remove the quotes and escape the whitespace characters to keep it as one string and also escape characters that could be considered as regex special characters.